app/src/main/java/com/minimalisticapps/priceconverter/presentation/home/HomeScreen.kt

```diff
@@ -73,17 +66,6 @@ fun HomeScreen(
     if (coinsState.isLoading && isRefreshing == false)
         ShowProgressDialog()
 
-    // Default usd currency added once app launch
-    if (!PCSharedStorage.isUsdAddDefault()) {
```

Comment:
@hafizsaadiftikhar This is probably mistake, without this, the default  USD is not there when app starts

Reply:
@hank-milliken Yes, It was removed by mistake while merging. I had already added. Don't worry about it.